skosify/infer.py

```diff
@@ -24,7 +24,11 @@ def skos_topConcept(rdf):
 
 
 def skos_hierarchical(rdf, narrower=True):
-    """Infer skos:broader/skos:narrower (S25) but only keep skos:narrower on request."""
+    """Infer skos:broader/skos:narrower (S25) but only keep skos:narrower on request.
+
+    :param bool narrower: If set to False, skos:narrowMatch will not be added,
```

Comment:
Shouldn't this be "skos:narrower will not be added"?

Reply:
Good catch! Fixed